package.json

```diff
@@ -22,11 +22,13 @@
     "build": "npm run compile",
     "clean": "rm -rf ./build/",
     "compile": "tsc -p . --rootDir . --outDir build/",
+    "format-check": "echo \"TODO\"",
```

Comment:
Do we really need a format check?  Or should we just go ahead and auto-format on build?